lib/maze/capabilities.rb

```diff
@@ -36,13 +36,16 @@ def for_browser_stack_browser(browser_type, local_id, capabilities_option)
       end
 
       # @param device_type [String]
-      def for_bitbar_device(bitbar_api_key, device_type, capabilities_option)
+      def for_bitbar_device(bitbar_api_key, device_type, platform, platform_version, capabilities_option)
         capabilities = {
           'bitbar_apiKey' => bitbar_api_key,
-          'bitbar_testrun' => device_type,
-          'bitbar_findDevice' => false
+          'bitbar_testrun' => "#{platform} #{platform_version}",
+          'bitbar_findDevice' => false,
+          'bitbar_testTimeout' => 3600,
```

Comment:
Is this the maximum allowed?  Some test suites take over an hour to run (if we run then all on the same device).  Can this be bumped to 1.5 or even 2 hours.